lib/block-supports/duotone.php

```diff
@@ -459,20 +459,30 @@ function gutenberg_render_duotone_support( $block_content, $block ) {
 	wp_add_inline_style( $filter_id, $filter_style );
 	wp_enqueue_style( $filter_id );
 
-	// Render any custom filter the user may have added.
 	add_action(
-		// There are a couple of known rendering quirks in Safari.
-		// 1. Filters won't render at all when the SVG is in the head of
-		// the document.
-		// 2. Filters display incorrectly when the SVG is defined after
-		// where the filter is used in the document, so the footer does
-		// not work.
-		'wp_body_open',
+		is_admin() ? 'admin_footer' : 'wp_footer',
```

Comment:
Why the check for `is_admin`. Isn't this filter only needed in the frontend?

Reply:
That was how it was prior to `wp_body_open`, but you're right, the filter is only needed for the frontend now.